java/project/README.md

```diff
@@ -1,20 +1,31 @@
 # Project
-Being able to project between spatial references is fundamental to a GIS. An example of when you would need to re-project data is if you had data in two different spatial references, but wanted to perform an intersect analysis with the `GeometryEngine::intersect` function. This function takes two geometries as parameters, and both geometries must be in the same spatial reference. If they are not, you could first use `GeometryEngine::project` to convert the geometries so they match.
 
-![Project App](project.png)
+Project a point from one spatial reference to another.
+
+![Image of project](project.png)
+
+## Use case
+
+Being able to project between spatial references is fundamental to a GIS. An example of when you would need to re-project data is if you had data in two different spatial references, but wanted to perform an intersect analysis with the `GeometryEngine.intersect` function. This function takes two geometries as parameters, and both geometries must be in the same spatial reference. If they are not, you could first use `GeometryEngine.project` to convert the geometries so they match.
```

Comment:
Is it meant to be "a GIS"? I'm unfamiliar with how people talk about GIS 😅

Reply:
yup, that's ok to say: think of it as "is fundamental to a geographic information system".